src/app/components/Amount/AmountCrypto.tsx

```diff
@@ -24,7 +24,7 @@ const AmountCrypto: React.FC<AmountProperties> = ({
 	}
 
 	return (
-		<span data-testid="AmountCrypto" className={cn("whitespace-nowrap", className)}>
+		<span data-testid="AmountCrypto" className={cn("space-x-1 whitespace-nowrap", className)}>
```

Comment:
```suggestion
		<span data-testid="AmountCrypto" className={cn("whitespace-nowrap", className)}>
```
i think this is a leftover code
`formattedAmount` is string

Reply:
it is, yea. was testing some things :hide: